program/src/matching.rs

```diff
@@ -1026,7 +1029,15 @@ impl<'a> Book<'a> {
         if post_allowed {
             // price limit check computed lazily to save CU on average
             let native_price = market.lot_to_native_price(price);
-            if native_price.checked_div(oracle_price).unwrap() > info.maint_liab_weight {
+
+            // Temporary hard coding LUNA price limit for bid to be below 10c.
+            // This is safe because it's already in reduce only mode
+            if market_index == LUNA_MARKET_INDEX && native_price >= market.lot_to_native_price(10) {
```

Comment:
Could you explain why this is needed? If the oracle is below 9 the normal price check would cover it. So this is to guard against weird oracle data?

Reply:
Ah, I think I get it. The existing price limit was too strict given the erratic oracle. This actually makes it more permissive.